# Dynamic Planner Default Params
main_run_freq                           : 2.5
voxel_dim                               : 0.2   # Unit: meter
robot_dim                               : 0.8   # Unit: meter
vehicle_height                          : 0.5   # Unit: meter
sensor_range                            : 30.0  # Unit: meter
reproject_dist                          : 5.0   # Unit: meter
layer_resoltion                         : 2.0
neighbor_layer_num                      : 3
visualize_ratio                         : 0.75
is_inter_navpoint                       : true
is_simulation                           : true
is_trajectory                           : false
attempt_autoswitch                      : true
world_frame                             : map

# Graph Messager
GraphMsger/robot_id                     : 0

# Map Handler Params
MapHandler/ceil_length                  : 10.0    # Unit: meter
MapHandler/map_grid_max_length          : 2000.0 # Unit: meter
MapHandler/map_grad_max_height          : 100.0  # Unit: meter 

# Scan Handler Params
ScanHandler/inflate_scan_size           : 2

# Dynamic Planner Utility Params
Util/angle_noise                        : 12.0 # Unit: degree
Util/accept_max_align_angle             : 12.0  # Unit: degree
Util/new_intensity_thred                : 2.0
Util/nav_clear_dist                     : 0.75
Util/terrain_free_Z                     : 0.25
Util/dyosb_update_thred                 : 4
Util/new_point_counter                  : 3
Util/inflate_counter                    : 1
Util/dynamic_obs_dacay_time             : 10.0

# Dynamic Graph Params
Graph/node_near_dist                    : 1.0   # Unit: meter
Graph/move_thred                        : 0.1   # Unit: meter
Graph/connect_votes_size                : 10
Graph/trajectory_interval_ratio         : 2.5
Graph/terrain_inflate_size              : 2      
Graph/clear_dumper_thred                : 4
Graph/node_finalize_thred               : 6
Graph/RANSAC_pool_size                  : 12
Graphj/vertical_angle_thred             : 30.0  # Unit: degree
Graph/connect_angle_thred               : 6.0   # Unit: degree
Graph/pos_filter_margin                 : 0.5   # Unit: meter
Graph/dirs_filter_margin                : 10.0  # Unit: degree

# Corner Detector Params
CDetector/resize_ratio                  : 5.0
CDetector/img_blur_size                 : 15.0
CDetector/filter_count_value            : 7
CDetector/is_inflate_image              : true
CDetector/is_save_img                   : true
CDetector/img_folder_path               : /media/bottle/hbt_ws/AirFar/data/IMAGES/forest/

# Graph Planner Params
GPlanner/converge_distance              : 0.5
GPlanner/goal_adjust_radius             : 5.0
GPlanner/goal_z_tolerance               : 2.0
GPlanner/free_counter_thred             : 7
GPlanner/free_box_dim                   : 0.85 # Unit: meter
GPlanner/reach_goal_vote_size           : 5
GPlanner/move_momentum_dist             : 0.85
GPlanner/path_momentum_thred            : 5
GPlanner/clear_inflate_size             : 3

# Contour Graph Params
ContourGraph/around_distance            : 0.75
ContourGraph/pillar_perimeter_dist      : 3.0  # Unit: meter


